assume/markets/clearing_algorithms/complex_clearing.py

```diff
@@ -341,22 +360,24 @@ def clear(
 
 def calculate_order_surplus(order, market_clearing_prices, instance, children):
     """
-    Calculate the surplus of an order given the market clearing prices and results of the market clearing.
+    Calculates the surplus of an order given the market clearing prices and results of the market clearing.
 
     Args:
         order (dict): The order
-        market_clearing_prices (dict): The market clearing prices
-        instance (pyomo.ConcreteModel): The solved pyomo model containing the results of the market clearing
-        children (list[dict]): The linked child bids of the given order
+        market_clearing_prices (dict): The market clearing prices.
+        instance (pyomo.ConcreteModel): The solved pyomo model containing the results of the market clearing.
+        children (list[dict]): The linked child bids of the given order.
```

Comment:
I do not think punctuation is required here, but I don't mind :smile:

Reply:
I would rather not have any punctuation in the args sections - as it is not a real sentence anyway :smile: 
But thats probably the least important thing